test/it/fixtures/test.suite.ts

```diff
@@ -46,6 +46,9 @@ class Basic {
             setTimeout(() => reject(new Error("Ooopsss...")), 1);
         });
     }
+
+    public classesMayHaveNonTestMethods() {
```

Comment:
I think that this is already covered. Suites that do not provide any tests will be ignored.